### ISSUE TITLE
New Question "Custom" page does not load when google-analytics.com is blocked

### ISSUE THREAD

**Issue by invadergir** [metabase/metabase#7568]
If you have firewall or script blocker (ie. NoScript/ScriptSafe) set to block access to google-analytics.com, the "Custom" New Question page will not load.  The "Select a table" cell has a "Loading" animation that spins forever and you can't create a question.  (See screenshot.)  

Tested on:

Chromium 66.0.3359.117
Firefox ESR 52.7.3
metabase jar v0.29.0
openjdk 8

![metabase-new-question](https://user-images.githubusercontent.com/24591824/39715610-7ce75698-51f3-11e8-99ea-2f4edb93b96a.png)

**Comment by salsakran** [comment]
is there anything in the javascript console?

**Comment by invadergir** [comment]
When I refresh the page with scripts disabled, I see these:

pulse:58 GET https://ajax.googleapis.com/ajax/libs/webfont/1.4.7/webfont.js 0 ()
loadScript @ pulse:58
(anonymous) @ pulse:60
(anonymous) @ pulse:69
pulse:75 GET https://www.google-analytics.com/analytics.js 0 ()
(anonymous) @ pulse:75
(anonymous) @ pulse:76

**Comment by philip-n** [comment]
Had the same problem on multiple browsers (Firefox, Chrome), multiple computers, using either the included sample dataset or an SQL Server database. Happened with Metabase 0.28.6 and 0.29. No adblock activated, but our corporate firewall might block some stuff (?).

Problem was always there after a new installation, and "playing around" seemed to help:
- Just now (metabase v0.29, Chrome 66.0.3359), I got rid of the problem by [Ctrl] + [F5]ing the offending screen.
- Previously, adding a new metric and creating a question based on this metric seems to have helped.

After the problem has disappeared once, it seems to disappear consistently.

Sorry for the vagueness, I'll add details if it happens again.

**Comment by mathieufortin01** [comment]
CTRL-F5 fixed the problem for me after a new install.

**Comment by flamber** [comment]
I cannot reproduce this issue on 0.32.9, but there has been several changes to how scripts are loaded.

**Comment by salsakran** [comment]
@tlrobinson my hunch is that this is no longer a problem but can we confirm and if so close this?

**Comment by tlrobinson** [comment]
I've recently been running an ad blocker which blocks google-analytics.com and I've never seen this during development or on stats, so I'm going to go ahead and close it since no one has reported it in more than a year.

@invadergir @philip-n @mathieufortin01 if you run into it again we can re-open this.
